templatefunc.go

```diff
@@ -352,11 +358,25 @@ func parseFormToStruct(form url.Values, objT reflect.Type, objV reflect.Value) e
 		case reflect.Struct:
 			switch fieldT.Type.String() {
 			case "time.Time":
-				format := time.RFC3339
-				if len(tags) > 1 {
-					format = tags[1]
+				var (
+					t   time.Time
+					err error
+				)
+				if len(value) >= 19 {
+					value = value[:19]
+					t, err = time.ParseInLocation(formatDateTime, value, time.Local)
```

Comment:
Maybe I take this too simple, You can see I was copy the codes from beego/orm. 
If it's only parsing time from html form, I think the three format would be enough.